Standards/Standards/Eurocode/EN1990.cs

```diff
@@ -5,13 +5,19 @@ namespace MagmaWorks.Taxonomy.Standards.Eurocode
     public class EN1990 : IEurocode
     {
         public StandardBody Body => StandardBody.EN;
-        public Enum Code => Eurocode.EN1990;
+        public Enum Code { get; } = Eurocode.EN1990;
         public Enum Part => EN1990Part;
         public NationalAnnex NationalAnnex { get; set; }
         public string Title => GetTitle();
 
         private EN1990Part EN1990Part = EN1990Part.None;
 
+        public EN1990()
+        {
+            Code = Eurocode.EN1990;
+            NationalAnnex = NationalAnnex.RecommendedValues;
```

Comment:
You could just initialise the properties directly and leave the default constructor empty

Reply:
Fixed with https://github.com/MagmaWorks/Taxonomy/pull/31/commits/11425979e82f7f348997da4d51c85c689b09d17a